salt/_states/caasp_kubernetes_crds.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+
+
+def reconcile(name, namespace='kube-system', desired_resources=[]):
```

Comment:
I think this tries to be too smart too. A `caasp_kubernetes_secret.absent` and `caasp_kubernetes_secret.managed` would be more explicit and clear in my view.